### PR TITLE
test for has secure password

### PR DESCRIPTION
**Pull request by vicfriedman**
issue #11 @aviflombaum @randallreedjr

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
+require 'pry'
```
Comment (pletcher) [inline spec/models/user_spec.rb]: :fire:

**Comment by pletcher** [comment]
.learn file needs `- resources: []` and we should remove the `require 'pry'` line, but otherwise it's good to :ship:!

File: spec/models/user_spec.rb (hunk 1 above)
Comment (vicfriedman) [inline spec/models/user_spec.rb]: removed!

Comment (pletcher) [inline spec/models/user_spec.rb]: Hmmm, it looks like you updated the `wip` branch instead of `solution-wip`

Comment (vicfriedman) [inline spec/models/user_spec.rb]: yup - fixing `solution-wip` now

**Pull request merged**